unit/java_bytecode/java_bytecode_convert_method/convert_invoke_dynamic.cpp

```diff
@@ -24,6 +24,7 @@ struct lambda_assignment_test_datat
   irep_idt lambda_function_id;
 
   std::vector<exprt> expected_params;
+  bool should_return_value;
```

Comment:
Having been bitten in the past with uninit Booleans, I'd prefer an explicit default value

Reply:
Set to `true` as more likely to produce a failing test, reminding people to set it.